style.css

```diff
@@ -97,3 +97,312 @@
+}
+
+.skills {
+  font-family: 'Inter', sans-serif;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+  color: #3a4a42;
+  background-color: #ebf0ee;
+  padding: 8px 12px;
+  border-radius: 4px;
+}
+
+.see_button {
+  background-color: #36b37f;
+  border: none;
+  border-radius: 4px;
+  padding: 12px;
+  margin-bottom: 12px;
+  align-self: center;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  text-align: center;
+}
+
+.about_me {
+  padding-top: 72px;
+  background-image:
+    url('Assets/Illustrations/about me illustration 1.png'),
+    url('Assets/Illustrations/about me illustration 2.png');
+  background-repeat: no-repeat, no-repeat;
+  background-position-x: right, left;
+  background-position-y: top, 350px;
+}
+
+.pitch2 {
+  width: 90%;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.pitch_title {
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #172b4d;
+}
+
+.description {
+  font-family: 'Inter', sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #42526e;
+}
+
+.proficiency {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  margin-top: 3.3em;
+}
+
+.skill_card {
+  width: 90%;
+  height: 326px;
+  background-color: #ebf0ee;
+  margin: auto;
+  padding-top: 44px;
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+}
+
+.skill-card_icon {
+  background-color: #fff;
+  width: 80px;
+  height: 80px;
+  margin: auto;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  box-shadow: 0 16px 24px 0 #252f8914;
+}
+
+.skill-card_icon > img {
+  margin: auto;
+}
+
+.category {
+  font-family: 'Inter', sans-serif;
+  font-size: 24px;
+  font-weight: 600;
+  line-height: 32px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #344563;
+  margin: 32px 0 24px 0;
+}
+
+.skills_list2 {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 8px;
+}
+
+.capabilities {
+  font-family: 'Inter', sans-serif;
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: left;
+  color: #36b37e;
+  padding: 12px;
+  background-color: #fff;
+  border-radius: 8px;
+}
+
+.contact_me {
+  display: flex;
+  flex-direction: column;
+  gap: 38px;
+  padding: 56px 24px 32px 24px;
+  background-image: url('Assets/Illustrations/contact form illustration.png');
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-position-y: bottom;
+}
+
+.parting_shot {
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #172b4d;
+}
+
+.contact_form {
```

Comment:
[ OPTIONAL ]
Great work using underscore as a `classname` separator in your `CSS`, however, hyphen separators are widely accepted as the CSS classname separator because it ensures readability and consistency and has multiple support on different systems. 🚀 

- [ ] Please consider Hyphen separation in your future projects (example `contact-form`) 🤖